src/core/MeshBVH.js

```diff
@@ -230,12 +230,28 @@ export class MeshBVH {
 				let maxx = - Infinity;
 				let maxy = - Infinity;
 				let maxz = - Infinity;
+
 				for ( let i = 3 * offset, l = 3 * ( offset + count ); i < l; i ++ ) {
 
-					const index = indexArr[ i ] * stride + bufferOffset;
-					const x = posArr[ index + 0 ];
-					const y = posArr[ index + 1 ];
-					const z = posArr[ index + 2 ];
+					let x;
+					let y;
+					let z;
+
+					if ( posAttr.normalized ) {
```

Comment:
Accessing `posAttr.normalized` incurs a dictionary access every time it's read. It would be best to declare a local variable in the function scope once so it can be used later without accessing it from the original object:

```js
const normalized = posAttr.normalized;

for ( let i = 3 * offset, l = 3 * ( offset + count ); i < l; i ++ ) {

  // ...
  if ( normalize ) {

    // ...

  }  

}
```

Reply:
Done in new revision.

Reply:
Great, thanks! Can we do one more performance benchmark with the new changes to get a sense for how much they might have changed in the existing case?

Reply:
Sure, here are results from a few...looks like refit got _more_ expensive, but it did make BVH go back to regular performance (master copied from before, though I did do a verification benchmark to make sure that the numbers hadn't changed notably)
```
# NEW VERSION
            CENTER / AVERAGE / SAH
BVH         96 / 96 / 740    
Refit       11.9 / 12.0 / 11.9
Refit/Hint  0.5 / 3.0 / 0.4

# MASTER
BVH         95 / 94 / 736
Refit       9.3 / 9.2 / 9.5
Refit/Hint  0.4 / 2.4 / 0.3
```

Reply:
That's interesting - the BVH construction looks good, now, but I'm surprise refit went up. Any thoughts on why that might have happened?

Reply:
At the time, I had no idea, and so shared those numbers in hopes you might have some insights.

However, I did some extra benchmarking...if I replace the `if (normalized)` line (line 239 in MeshBVH.js in the current version of the PR) with `if (false)` and run the benchmark, I get ~9.4ms refit times again. The normalized variable is set to false (and I confirmed it as so) in all of the benchmarks, so it's not like it's going down the wrong code path. It appears just checking for the normalized variable in that `if` statement is adding that time.

Other things I checked:
1. If I create a random variable `test = false`, defined in the block where I define `normalize` and replace that if statement check with `if (test)`, it still takes 12ms. This was true for pretty much every combination of variables, including silly ones like using the existing variables and checking to see if they were `true`.
2. Using the getter all the time without the normalize check gives me ~10.5ms per refit

I'm surprised that the `if` statement would cause such a difference -- though I suppose refit does a lot of operations for how little time it takes, and every operation adds up. What are your thoughts on how we should proceed here?

Reply:
Yeah these kinds of tight loops are always sensitive to even the smallest logic addition. I think any other solution will have too much redundant logic to be worth it.

Do you mind just making a new issue mentioning that the performance degraded a little bit with this PR as well as why and the relative difference? I think the feature improvement of supporting normalized buffers is worth it but it would be nice to keep track of the known perf loss in case there are other thoughts on how to improve it later on.

Reply:
I can make the issue, but I actually think with those numbers it might be better to skip the if statement and always use the getters, since that was less of a performance regression on my machine than adding in the if statement. I don't know if this will hold true for other machines, though.

Reply:
> it might be better to skip the if statement and always use the getters, since that was less of a performance regression on my machine than adding in the if statement.

I think I agree with that. Let me run some tests on my machine when I get a chance before we commit to that, though.

Reply:
Yeah I'm seeing that the refit perf is better with just the getters:

Before (with branch):
~15.517233 ms

After (just getters):
~11.974284 ms

This is on a gen 2 M1 macbook pro. So lets go ahead and switch back to just the getters branch.